brunch-config.js

```diff
@@ -28,5 +34,14 @@ module.exports = {
         })
       ]
     }
+  },
+  sourceMaps: 'inline', // For karma-sourcemap-loader.
+  overrides: {
+    production: {
+      paths: {
+        // Exclude files in "test" from compilation.
+        watched: ['app', 'vendor']
```

Comment:
To answer both of these questions, you would use a `vendor` directory to store non-CommonJS scripts and/or scripts that aren't on NPM. See [Brunch: Core Concepts](http://brunch.io/docs/concepts) and [the Brunch Guide](https://github.com/brunch/brunch-guide/blob/master/content/en/chapter03-conventions-and-defaults.md#folders) and search for "vendor" for details.